tests/nonbonded/test_nonbonded_interaction_group.py

```diff
@@ -116,7 +116,7 @@ def ref_ixngroups(conf, params, box, lamb):
         conf,
         params,
         example_box,
-        lamb=lamb,
+        lamb,
```

Comment:
Style preference or functional?

Reply:
The name in there has to match the positional name. I disliked the coupling and is also redundant.